src/parser/tests.rs

```diff
@@ -544,20 +581,15 @@ fn test_global_stmts() {
     setup_logger();
     let mut err = ErrorGen::new("".to_string(), "".to_string(), 0);
     let script = r#"
-        i32 a;
-        a = 1;
         dummy_fn() {
             a = strcmp((arg0, arg1), "bookings");
             strcmp((arg0, arg1), "bookings");
         }
-        BEGIN{
+        wasm::call:alt{
+            (i32, i32) a = (arg0, arg1);
             strcmp((arg0, arg1), "bookings");
         }
-        END {
-            a = 2;
-        }
     "#;
-
```

Comment:
It doesn't look like this is actually testing global statements anymore, maybe add as a new test instead?